source/fundamentals/aggregation-expression-operations.txt

```diff
@@ -0,0 +1,1211 @@
+     - :manual:`$anyElementTrue </reference/operator/aggregation/anyElementTrue/>`
+
+   * - `concat() <{+api+}/apidocs/mongodb-driver-core/com/mongodb/client/model/mql/MqlArray.html#concat(com.mongodb.client.model.mql.MqlArray)>`__
+     - :manual:`$concatArrays </reference/operator/aggregation/concatArrays/>`
+
+   * - `concatArrays() <{+api+}/apidocs/mongodb-driver-core/com/mongodb/client/model/mql/MqlArray.html#concatArrays(java.util.function.Function)>`__
+     - :manual:`$concatArrays </reference/operator/aggregation/concatArrays/>`
+
+   * - `contains() <{+api+}/apidocs/mongodb-driver-core/com/mongodb/client/model/mql/MqlArray.html#contains(T)>`__
+     - :manual:`$in </reference/operator/aggregation/in/>`
+
+   * - `distinct() <{+api+}/apidocs/mongodb-driver-core/com/mongodb/client/model/mql/MqlArray.html#distinct()>`__
+     - :manual:`$setUnion </reference/operator/aggregation/setUnion/>`
+
+   * - `elementAt() <{+api+}/apidocs/mongodb-driver-core/com/mongodb/client/model/mql/MqlArray.html#elementAt(int)>`__
+     - :manual:`$arrayElemAt </reference/operator/aggregation/arrayElemAt/>`
+
+   * - `filter() <{+api+}/apidocs/mongodb-driver-core/com/mongodb/client/model/mql/MqlArray.html#filter(java.util.function.Function)>`__
+     - :manual:`$filter </reference/operator/aggregation/filter/>`
+
+   * - `first() <{+api+}/apidocs/mongodb-driver-core/com/mongodb/client/model/mql/MqlArray.html#first()>`__
+     - :manual:`$first </reference/operator/aggregation/first-array-element/>`
+
+   * - `joinStrings() <{+api+}/apidocs/mongodb-driver-core/com/mongodb/client/model/mql/MqlArray.html#joinStrings(java.util.function.Function)>`__
+     - :manual:`$concat </reference/operator/aggregation/concat/>`
+
+   * - `last() <{+api+}/apidocs/mongodb-driver-core/com/mongodb/client/model/mql/MqlArray.html#last()>`__
+     - :manual:`$last </reference/operator/aggregation/last-array-element/>`
+
+   * - `map() <{+api+}/apidocs/mongodb-driver-core/com/mongodb/client/model/mql/MqlArray.html#map(java.util.function.Function)>`__
+     - :manual:`$map </reference/operator/aggregation/map/>`
+
+   * - `max() <{+api+}/apidocs/mongodb-driver-core/com/mongodb/client/model/mql/MqlArray.html#max(T)>`__
+     - :manual:`$max </reference/operator/aggregation/max/>`
+
+   * - `maxN() <{+api+}/apidocs/mongodb-driver-core/com/mongodb/client/model/mql/MqlArray.html#maxN(com.mongodb.client.model.mql.MqlInteger)>`__
+     - :manual:`$maxN </reference/operator/aggregation/maxN-array-element/>`
+
+   * - `min() <{+api+}/apidocs/mongodb-driver-core/com/mongodb/client/model/mql/MqlArray.html#min(T)>`__
+     - :manual:`$min </reference/operator/aggregation/min/>`
+
+   * - `minN() <{+api+}/apidocs/mongodb-driver-core/com/mongodb/client/model/mql/MqlArray.html#minN(com.mongodb.client.model.mql.MqlInteger)>`__
+     - :manual:`$map </reference/operator/aggregation/minN-array-element/>`
```

Comment:
minN should map to minN

and I think the map method is missing

Reply:
https://docs-mongodbcom-staging.corp.mongodb.com/java/docsworker-xlarge/docsp-27390-aggregation-operations/fundamentals/aggregation-expression-operations/#array-operations:~:text=%24last-,map(),-%24map

Here?